includes/admin/class-amp-plugins.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Class AMP_Plugins
+ *
+ * @package AMP
+ * @since   2.2.2
+ */
+
+/**
+ * Class to add
+ */
+class AMP_Plugins {
```

Comment:
When adding new classes, we should use the `Services` architecture going under `src/`.

Nevertheless, instead of adding a whole new class the new logic can just go into the existing `PluginSuppression` class.